rest-api-spec/src/main/resources/rest-api-spec/api/fleet.msearch.json

```diff
@@ -0,0 +1,39 @@
+{
+  "fleet.msearch":{
+    "documentation":{
+      "url": null,
+      "description": "Multi Search API where the search will only be executed after specified checkpoints are available due to a refresh. This API is designed for internal use by the fleet server project."
+    },
+    "stability":"experimental",
+    "visibility":"public",
+    "headers":{
+      "accept": [ "application/json"],
+      "content_type": ["application/json"]
```

Comment:
Since this is a bulk body this should be `content_type: ["application/x-ndjson"]`